src/rocker/templates/bashrc_extensions_user_snippet.Dockerfile.em

```diff
@@ -0,0 +1,6 @@
+WORKDIR @home_dir
+RUN echo "\n# Source custom bashrc extensions" >> @home_dir/.bashrc
+@[for path, filename in bashrc_extension_files.items()]
+COPY @path @filename
```

Comment:
This is effectively a temporary file, it should probably go into somewhere like /tmp and not potentially override content in their home directory. (For example if someone tries to append their own local `.bashrc` it might get messy.